src/test/java/org/folio/rest/impl/FinanceDataApiTest.java

```diff
@@ -128,6 +135,68 @@ void negative_testGetFinanceFinanceDataFailure() {
     assertThat(errors.getErrors().get(0).getCode(), is(GENERIC_ERROR_CODE.getCode()));
   }
 
+  @Test
```

Comment:
If Copilot is used please add annotation, we have ask to marker everything that generated with help of copilot

Reply:
Added Copilot generated mark